### PR TITLE
IGAPP-1005: Removed double separation line in location list view mobile

### PR DESCRIPTION
**Pull request by f1sh1918**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

**Comment by Cmd190** [comment]
Tested on chrome, edge and firefox on Windows. The double seperation line is still shown on "middle" window size in firefox. See screenshot below.
 
image
(left: chrome; right: firefox)

**Review by SteffiStuffel — APPROVED** [review]
Tested on chrome, double line is not shown. :tada:

File: release-notes/unreleased/IGAPP-1005-double-separation-line-poi-list-bas.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+issue_key: IGAPP-1005
+show_in_stores: true
+platforms:
+  - web
+en: Removed double separation line in location list view mobile.
+de: Doppelte Trennlinie in der mobilen Listenansicht der Lokationen entfernt.
```
Comment (steffenkleinle) [inline release-notes/unreleased/IGAPP-1005-double-separation-line-poi-list-bas.yml:6]: Lokationen is definitely not a term we use:
```suggestion
en: Removed double separation line in location list view mobile.
de: Doppelte Trennlinie in der mobilen Listenansicht der Orte entfernt.
```

**Review by steffenkleinle — CHANGES_REQUESTED** [review]
Can reproduce the issue @Cmd190 described, for 972x720px.

**Comment by f1sh1918** [comment]
> Can reproduce the issue @Cmd190 described, for 972x720px.

well the purpose of this ticket @steffenkleinle as you created was to remove:
"Double separation line in bottom action sheet"

**Review by steffenkleinle — APPROVED** [review]

**Pull request merged**